sdk/tests/conformance2/rendering/attrib-type-match.html

```diff
@@ -115,11 +216,8 @@
     }
     wtu.glErrorShouldBe(gl, gl.NO_ERROR, "No GL error from set up");
 
-    debug("");
-    debug("Test vertexAttrib with drawArrays and drawArraysInstanced");
     wtu.setupUnitQuad(gl, 0);
 
-    debug("Correct setup");
```

Comment:
why do you remove this line?  I think this indicates the following test is a positive test, whereas all other cases are negative test.  This line makes the test case clearer.